### PR TITLE
NCDA scoreboard - Child Identification pane

### PR DESCRIPTION
**Pull request by anvmn**
#525

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1204

**Comment by anvmn** [comment]
Selection_1205

**Comment by anvmn** [comment]
Selection_1206

**Comment by anvmn** [comment]
@AronNovak please review

File: client/src/elm/Translate.elm [hunk 1]
```diff
@@ -4723,8 +4749,13 @@ translationSet trans =
                     , kinyarwanda = Nothing
                     }
 
-        FatherName ->
-            { english = "Father's Name"
+        FatherOrCheifId ->
+            { english = "Father or Cheif of Family ID"
```
Comment (AronNovak) [inline client/src/elm/Translate.elm]:
```suggestion
            { english = "Father or Chief of Family ID"
```

File: client/src/elm/Translate.elm [hunk 2]
```diff
@@ -4723,8 +4749,13 @@ translationSet trans =
                     , kinyarwanda = Nothing
                     }
 
-        FatherName ->
-            { english = "Father's Name"
+        FatherOrCheifId ->
```
Comment (AronNovak) [inline client/src/elm/Translate.elm]:
```suggestion
        FatherOrChiefId ->
```

File: client/src/elm/Translate.elm [hunk 3]
```diff
@@ -4723,8 +4749,13 @@ translationSet trans =
...
+            { english = "Father or Cheif of Family ID"
+            , kinyarwanda = Nothing
+            }
+
+        FatherOrCheifName ->
```
Comment (AronNovak) [inline client/src/elm/Translate.elm]:
```suggestion
        FatherOrChiefName ->
```

File: client/src/elm/Translate.elm [hunk 4]
```diff
@@ -4723,8 +4749,13 @@ translationSet trans =
...
+            { english = "Fathers or Cheif of Family Name"
```
Comment (AronNovak) [inline client/src/elm/Translate.elm]:
```suggestion
            { english = "Fathers or Chief of Family Name"
```

Comment (AronNovak) [reply client/src/elm/Translate.elm]: https://en.wiktionary.org/wiki/cheif - maybe on purpose?

**Review by AronNovak — APPROVED** [review]

File: client/src/elm/Translate.elm (hunk 4 above)
Comment (anvmn) [reply client/src/elm/Translate.elm]: Just a typo. Thanks @AronNovak .

**Pull request merged**